x/rewards/types/genesis.go

```diff
@@ -31,5 +35,20 @@ func (gs GenesisState) Validate() error {
 			at all times. amount of timers found: %v`, len(timeEntries))
 	}
 
+	for _, sub := range gs.IprpcSubscriptions {
+		_, err := sdk.AccAddressFromBech32(sub)
+		if err != nil {
+			return fmt.Errorf("invalid subscription address. err: %s", err.Error())
+		}
+	}
+
+	if gs.MinIprpcCost.Denom != DefaultGenesis().MinIprpcCost.Denom {
```

Comment:
this is the same as  if gs.MinIprpcCost.Denom != commontypes.Bonddenom.
either take from staking module or do it explicitly

Reply:
done. see https://github.com/lavanet/lava/commit/dc29e1a1e0181b1110dac3e10733bc10915a321e